feature-libs/product-configurator/rulebased/components/attribute/types/multi-selection-bundle/configurator-attribute-multi-selection-bundle.component.ts

```diff
@@ -111,21 +120,14 @@ export class ConfiguratorAttributeMultiSelectionBundleComponent
   }
 
   onSelect(eventValue: any): void {
```

Comment:
change `updateMultipleSelectionValues` to return `multipleSelectionValues`, and in this function call `super.onSelect(this.updateMultipleSelectionValues(eventValue, true))`. Same for the `onDeselect`.